pkg/maps/configmap/doc.go

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+// Package config stores runtime configuration state for the Cilium datapath.
+// +groupName=maps
```

Comment:
Why is groupName here? AFAIK it only affects code generation for k8s object structs and typed clients.

Reply:
IDK, all the packages under `pkg/maps` have this `groupName=maps`.